src/content/en/2021/accessibility.md

```diff
@@ -220,15 +221,15 @@ Before the introduction of HTML5, ARIA landmark roles were needed to accomplish
     </tbody>
   </table>
   <figcaption>{{ figure_link(
-    caption="Landmark elment and role usage (desktop).",
+    caption="Landmark element and role usage (desktop).",
     sheets_gid="1736231238",
     sql_file="landmark_elements_and_roles.sql"
   ) }}</figcaption>
 </figure>
 
-The most commonly expected landmarks that the majority of web pages should have are `<main>`, `<header>`, `<nav>` and `<footer>`. We found that only 28% of desktop pages have a native HTML `<main>` element, and 17% of desktop pages have an element with a `role="main"`, and 35% of pages have either.
+The most commonly expected landmarks that the majority of web pages should have, are `<main>`, `<header>`, `<nav>` and `<footer>`. We found that only 28% of desktop pages have a native HTML `<main>` element, and 17% of desktop pages have an element with a `role="main"`, and 35% of pages have either.
```

Comment:
can we remove the first instance of the word "and" here: 
"native HTML `<main>` element, **and** 17% of desktop pages have an element with a `role="main"`, and 35% of pages have either."